docs/reference/actions.md

```diff
@@ -1,3 +1,4 @@
 # Actions
 
-See [Actions](https://charmhub.io/discourse-k8s/actions).
\ No newline at end of file
+See [Actions](https://charmhub.io/discourse-k8s/actions).
+> Read more about actions in the Juju docs: [Action](https://juju.is/docs/juju/action)
```

Comment:
There is no blank line between line 3 and 4 here, but there is for the `docs/reference/configurations.md`.
See below comment.